test/regression_tests.py

```diff
@@ -11,26 +12,77 @@
 from ts_scripts.api_utils import test_api
 from ts_scripts.install_from_src import install_from_src
 from ts_scripts.regression_utils import test_regression
-from ts_scripts.utils import check_python_version
+from ts_scripts.utils import check_python_version, try_and_handle
 
-now = datetime.datetime.now()
-print("Current date and time : " + now.strftime("%Y-%m-%d %H:%M:%S"))
 
-check_python_version()
+def regression_tests(binaries, pypi, conda, nightly):
+    now = datetime.datetime.now()
+    print("Current date and time : " + now.strftime("%Y-%m-%d %H:%M:%S"))
 
-# Install from source
-install_from_src()
+    check_python_version()
 
-# Generate mar file
-mg.generate_mars()
+    if binaries:
+        if pypi:
+            if nightly:
+                cmd = f"pip install torchserve-nightly torch-model-archiver-nightly torch-workflow-archiver-nightly"
+            else:
+                cmd = f"pip install torchserve torch-model-archiver torch-workflow-archiver"
+        elif conda:
+            if nightly:
+                cmd = f"conda install -c pytorch-nightly torchserve torch-model-archiver torch-workflow-archiver"
+            else:
+                cmd = f"conda install -c pytorch torchserve torch-model-archiver torch-workflow-archiver"
+        try_and_handle(cmd, False)
```

Comment:
Is there any value in running the command with try_and_handle here? The only exception it handles, its handling by throwing it again, and we do not use dry_run here, so why not call the command directly?